pages/common/pip.md

```diff
@@ -8,9 +8,17 @@
 
 `pip install {{package_name}}`
 
+- Install a specific version of a package:
+
+`pip install {{package}}=={{version}}`
+
+- Install a package to the user's directory instead of the system-wide default location:
```

Comment:
This example was removed in #4983 and moved to `pip-install`.
```suggestion
- Install a package to the user's directory instead of the system-wide default location:
```